devops/ansible/roles/php70/vars/main.yml

```diff
@@ -9,7 +9,7 @@ debug_mode: false
 # allow_url_fopen
 #   Default Value: On
 php_allow_url_fopen: "Off"
-php_disable_functions: "exec,passthru,shell_exec,system,proc_open,popen"
+php_disable_functions: "exec,passthru,shell_exec,system,popen"
```

Comment:
I'd prefer using `wp-cli` for consistency sake.

Can you research enable the `proc_` functions just for wp-cli? 

I may be overstating the security risk. If that's the case, enable it and add a comment here noting why it's not included in the list of disabled functions.